### DISCUSSION TITLE
Need help regarding database migration

### DISCUSSION THREAD

**Discussion by saunish** [loopbackio/loopback-next#7735] [Q&A] [answered]
getting this error "Cannot migrate models not attached to this datasource"

I have added one model and unable to migrate also have added 

await app.migrateSchema({existingSchema, models: ['User']});

**Comment by sohaibahsan007** [comment, marked as answer]
Hi, 
we can talk and figure this out. can you hit me on loopback4 slack channel?

**Reply by saunish** [reply]
Thanks Sohaib, this issue was solved by adding repository by using `lb4 repository`
